ebos/ecltracermodel.hh

```diff
@@ -154,11 +155,20 @@ public:
                 gas_.addTracer(tracerIdx, this->tracerConcentration_[tracerIdx]);
             }
         }
+        TracerMatrix* base = this->tracerMatrix_.get();
```

Comment:
Could you maybe add a comment here to remind the reader that `base` is guaranteed to remain valid even when moving from `tracerMatrix_`?

Reply:
done.